src/main/java/baritone/bot/utils/ExampleBaritoneControl.java

```diff
@@ -177,6 +178,10 @@ public void onSendChatMessage(ChatEvent event) {
                 }
             }
         }
+        if(msg.toLowerCase().equals("map")) {
+            MapChunk chunk = new MapChunk(world().getChunk(player().getPosition()));
+            chunk.writeImage();
```

Comment:
```
event.cancel();
return;
```

Reply:
Again, just for testing. There won't be a command in the final one.